src/gui/models/configurationModel.cpp

```diff
@@ -37,16 +37,24 @@ int ConfigurationModel::rowCount(const QModelIndex &parent) const
 
 QVariant ConfigurationModel::data(const QModelIndex &index, int role) const
 {
-    if (role == Qt::DisplayRole)
-        return QString::fromStdString(std::string(rawData(index)->name()));
-    else if (role == Qt::CheckStateRole && checkedItems_)
-        return std::find(checkedItems_->get().begin(), checkedItems_->get().end(), rawData(index)) == checkedItems_->get().end()
-                   ? Qt::Unchecked
-                   : Qt::Checked;
-    else if (role == Qt::UserRole)
-        return QVariant::fromValue(rawData(index));
-
-    return {};
+    switch (role)
+    {
+        case (Qt::DisplayRole):
+            return QString::fromStdString(std::string(rawData(index)->name()));
+        case (Qt::CheckStateRole):
+            if (checkedItems_)
+            {
+                return std::find(checkedItems_->get().begin(), checkedItems_->get().end(), rawData(index)) ==
+                               checkedItems_->get().end()
+                           ? Qt::Unchecked
+                           : Qt::Checked;
+            }
+            return {};
```

Comment:
```suggestion
            else { return {}; }
```

I find having the explicit `else` to be a bit clearer, since it makes the case statement a single expression.  However, this is purely a matter of taste and I'm open to disagreement.